### PR TITLE
pyexodus feedstock

### PR DESCRIPTION
**Pull request by krischer**
As the title says. Its a "pure Python" (still requires h5py) library to read and write Exodus mesh files.

https://github.com/SalvusHub/pyexodus

### CODE REVIEW

**Comment by krischer** [comment]
Ready to merge from my point of view. Thanks for all the work the maintainers do all the time :)

File: recipes/pyexodus/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # If getting the source from GitHub remove the line above
```
Comment (bgruening) [inline recipes/pyexodus/meta.yaml]: Can you please cleanup the recipe a little bit and remove the comments.
Thanks!

Comment (krischer) [reply recipes/pyexodus/meta.yaml]: Sure thing. Would you prefer to have all comments removed or only those that don't apply to this particular recipe? I actually think they are kind of helpful if one only occasionally looks at them.

Comment (bgruening) [reply recipes/pyexodus/meta.yaml]: All that do not really apply to this specific recipe. Its good to learn how things should be but in an official version they should imho be removed.

**Comment by krischer** [comment]
It cannot copy the license file - does anyone understand what I'm doing wrong?

**Comment by dougalsutherland** [comment]
You need to change the `about/license_file` entry: [see docs](https://conda-forge.org/docs/meta.html#packaging-the-license-manually).

Also, since this is your package, you should just add a `MANIFEST.in` file with the contents `include LICENSE.txt` for the next release; then it'll be in the pypi distribution and won't need to be included here.

**Comment by krischer** [comment]
Thanks for the help @dougalsutherland. It seems to work now and the next version of my package will include in its `MANIFEST.in`.

**Comment by dougalsutherland** [comment]
Looks good now. Thanks @krischer!

**Pull request merged**